Add passing GateKeeper example

willbeason opened the pull request:
This is a copy+paste of the example Max added to the konduct Go framework.

willbeason committed:
Add passing GateKeeper example

frankfarzan commented on gatekeeper.yaml:
```diff
@@ -0,0 +1,53 @@
+  - apiVersion: constraints.gatekeeper.sh/v1beta1
+    kind: K8sBannedConfigMapKeysV1
+    metadata:
+      name: no-secrets-in-configmap
```
Let's add a configmap that violates this

willbeason replied:
Done. Obviously the demo-runner now has to manually modify this to show the happy path.

willbeason committed:
Add violation to repository

Obviously this will no longer pass validation, so anyone using this in a demo has to manually modify it to fix it to show the "working" state.
